src/acl.c

```diff
@@ -2566,7 +2566,7 @@ int ACLSaveToFile(const char *filename) {
 
 /* This function is called once the server is already running, modules are
  * loaded, and we are ready to start, in order to load the ACLs either from
- * the pending list of users defined in redis.conf, or from the ACL file.
+ * the pending list of users defined in configuration file, or from the ACL file.
```

Comment:
There are grammatical errors being introduced into comments with this approach ('the' needs to proceed 'configuration file' now).

I really think these should all be a straight up redis.conf -> valkey.conf swap, otherwise the meaning is sometimes slightly changing (and when there are multiple configuration files, e.g. with an ACL file or a sentinel.conf, the intent is further confused).

Reply:
that is a good point, @natoscott. the brand-agnostic idea was very appealing to me but now reading through the actual changes with your call out, I can see the context getting lost. this might not be an issue for folks who work on the codebase on a daily basis (who probably don't read the comments much) but could be a barrier for the new member in the community. @zuiderkwast @madolson thoughts? should we revert back to valkey.conf?

Reply:
I'm convinced about valkey.conf. It's a safer and quicker replace, which is what we need right now.